### PR TITLE
LPS-117987 Fix the style of the error message

### PR DESCRIPTION
**Pull request by victorg1991**
This pr fixes a bug where error messages appeared on the screen with success styles. This was caused for using the wrong property to define the style when calling the openToast function.

Steps to reproduce (one of many ways):

- Go to applications menu
- Go to Content dashboard
- Click on the kebab menu and select configuration
- Leave the right box empty
- Save

### CODE REVIEW

File: util-taglib/src/com/liferay/taglib/ui/success/toast.tmpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Liferay.Util.openToast({
-   displayType: 'success',
+   display: 'success',
```
Comment (jbalsas) [inline util-taglib/src/com/liferay/taglib/ui/success/toast.tmpl:2]: Shouldn't this be `type` as well?

File: portal-web/docroot/html/taglib/ui/error/page.jsp [hunk 2]
```diff
@@ -48,7 +48,7 @@ String rowBreak = (String)request.getAttribute("liferay-ui:error:rowBreak");
 	<c:otherwise>
 		<aui:script>
 			Liferay.Util.openToast({
-			   displayType: '<%= alertStyle %>',
+			   type: '<%= alertStyle %>',
```
Comment (jbalsas) [inline portal-web/docroot/html/taglib/ui/error/page.jsp:51]: Sort is your friend :D

File: util-taglib/src/com/liferay/taglib/ui/success/toast.tmpl (hunk 1 above)
Comment (victorg1991) [reply util-taglib/src/com/liferay/taglib/ui/success/toast.tmpl:2]: mygod too many things in my head, sorry 🤪

**Comment by victorg1991** [comment]
Vaya desproposito de pr, fixed! 😅

**Comment by jbalsas** [comment]
> Vaya desproposito de pr, fixed! 😅

No worries at all @victorg1991!! Million thanks for sending this in!! ❤️ ❤️ ❤️

File: util-taglib/src/com/liferay/taglib/ui/success/toast.tmpl (hunk 1 above)
Comment (jbalsas) [reply util-taglib/src/com/liferay/taglib/ui/success/toast.tmpl]: chores

**Comment by jbalsas** [comment]
ci:test:sf

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:forward

**Pull request closed without merging**